schemas/components/schemas/VPNMethods.yaml

```diff
@@ -20,3 +21,7 @@ properties:
     type: boolean
     description: >-
       The browser runs on a different operating system than the operating system inferred from the request network signature.
+  relay:
+    type: boolean
+    description: >-
+      Request IP address belongs to a relay service provider.
```

Comment:
Can you give an example of a relay service we can detect? What is the difference between a relay and a VPN? VPN changes location and relay does not?

Reply:
I think This document can answer lot of your questions:
https://www.notion.so/fingerprintjs/Private-Relay-Detection-11902f125ebd80cdb829c36d7b6a131e?pvs=4

Let me know If you want me to add anything from here to the spec

Reply:
I am confused, are we detecting just Apple Private relay? Or also other "relay" services? Are we using  
"relay" as a general term for "VPN without location spoofing capabilities"?

Reply:
So, Relay is different from VPN ([link](https://www.notion.so/fingerprintjs/Private-Relay-Detection-11902f125ebd80cdb829c36d7b6a131e?pvs=4#13002f125ebd80bfb56ffa3ad179989a))

I think what you are trying to understand or question is product decisioning 

I can help you with my understanding

1. Relay detection (CloudFlare Warp, I-Cloud, Akamai, Fastly etc) has been asked by lot of our customers 
1.1 It's important to customers because relay's are mostly default provided configuration nowadays and they don't want to block such users because there intent is around privacy and not fraud
2. IpInfo provides us Relay detection for all these major providers and we are just abstraction over that
3. So Relay just means different level of anonymisation (but had to be grouped inside VPN to classify that the given virtual network is only `Relay` )

Reply:
Okay, I made a [suggestion here](https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/174/files#r1850224452), please let me know if its accurate based on your understanding and merge it if yes, thanks!